aksel.nav.no/website/pages/eksempler/switch/controlled-boolean.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import { Switch } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <Switch checked={checked} onChange={() => setChecked((x) => !x)}>
```

Comment:
Tanke: Hadde kanskje vært kult å vise at man kan gjøre `setChecked(event.target.checked)`? Vet i så fall ikke om det burde være i stedet for dette, eller et nytt eksempel.

Reply:
Tenkte ikke på den casen, men skal oppdatere da jeg tenker det er bedre praksis 👍

Reply:
Done ✅